### ISSUE TITLE
[Data Update]: University of Maryland, College Park: FY2025

### ISSUE THREAD

**Issue by dsmolyak** [CSStipendRankings/CSStipendRankings#121]
## Institution name: University of Maryland, College Park

- **Annual Stipend Amount (USD, both Pre-Qualification and Post-Qualification if different)**: pre-qual: 42,577; post-qual: 45,730 (9.5 month + 30 hours per week - see [previous issue](https://github.com/CSStipendRankings/CSStipendRankings/issues/69) for details)

- **Annual Local Living Wage (USD)**: 54,391

  - **Link to the [MIT Living Wage Calculator](http://livingwage.mit.edu/)**: https://livingwage.mit.edu/counties/24033

- **Annual Out-of-pocket Fees (and Health Insurance) Charged by University (USD)**: fees: $716 (1-8 credit semester charge of $358 * 2 semesters)

- **Summer Funding Guarantee**: No

- **Source of the stipend and fee data (e.g., your own data point, a link to an official website, etc.)**: [FY25 GA RATES COMP SCI.pdf](https://github.com/user-attachments/files/16550681/FY25.GA.RATES.COMP.SCI.pdf) for stipends, and [UMD website](https://billpay.umd.edu/GraduateTuition#:~:text=The%20University%20of%20Maryland%20%28UMD%29%20charges%20a%20fee,students%20in%20J-1%20status%29%20issued%20by%20the%20University.) for fees

- **Additional Comments (Optional)**:

**Comment by mjc0608** [comment]
Done

**Comment by dsmolyak** [comment]
Forgot to mention as well, international students at UMD do pay a CPT fee. PhD students who have completed coursework must sign up for a research credit, and there is no tuition remission or department coverage of this credit's cost (which is charged at an out-of-state rate of ~$2000 per credit.

More information available on the [UMD website](https://marylandglobal.umd.edu/global-learning-all/international-students-scholars/employment-taxes/employment-f-1-international-students/curricular-practical-training-cpt), particularly under "Step 2".

**Comment by mjc0608** [comment]
If this is the case, this should be counted into the fees. Can you update the number for fees?

**Comment by mjc0608** [comment]
> PhD students who have completed coursework must sign up for a research credit, and there is no tuition remission or department coverage of this credit's cost

This is basically a cut of stipend for students after qualification. Maybe we can just take that amount out of the stipend.

**Comment by dsmolyak** [comment]
Apologies, Phd students *who would like to do an internship over the summer* who have completed coursework must sign up for a research credit. Hence I'm hesitant to add that to the total fee count as only a subset of students fall within that criteria.

**Comment by mjc0608** [comment]
Yes, it should be counted into fees. Can you give me an updated total fee?

**Comment by mjc0608** [comment]
I found the rate here: https://billpay.umd.edu/GraduateTuition.
So the fee would be 716+1841=2557.
